src/cartridges/int_adyen_overlay/cartridge/scripts/util/adyenHelper.js

```diff
@@ -29,6 +29,8 @@ const constants = require('*/cartridge/adyenConstants/constants');
 const AdyenConfigs = require('*/cartridge/scripts/util/adyenConfigs');
 const Transaction = require('dw/system/Transaction');
 const UUIDUtils = require('dw/util/UUIDUtils');
+const jsonData = require('*/cartridge/scripts/config/blockedPaymentMethods.json'); 
```

Comment:
Better to use a specific name for the const rather than a generic name as `jsonData`